eva/binder/statement_binder.py

```diff
@@ -131,6 +133,8 @@ def _bind_select_statement(self, node: SelectStatement):
                 node.target_list = extend_star(self._binder_context)
             for expr in node.target_list:
                 self.bind(expr)
+        node.from_table = get_table_ref_subclass(node.from_table)
+        set_audio_video_tuple_flags(node)
```

Comment:
Move this logic to the end of this function. There might be column dependencies in `order_by` and `group_by` clause.